### ISSUE TITLE
Visit URL to Frontend in Headless mode

### ISSUE THREAD

**Issue by Buffalom** [statamic/ideas#122]
Hi everyone

I mostly use statamic headless and the visit url always points to the cms domain.
I haven't found a way to configure it to point to another domain in the frontend.
Would it be a big deal to implement a way to set the domain for the visit url? The visit url might not be the only one where this is relevant but it's the most obvious one that came to my mind.

I saw that the visit url is the permalink of the entry. I found a permalink on pretty much everything.
At first glance, to me it seems kind of difficult to add but obviously you guys have a better overview on everything.

Would be nice to hear you guys opinion on this.

Cheers

**Comment by Buffalom** [comment]
I just realised that I can set the sites url in `config/statamic/sites.php`.
At first I thought that might be wrong but now that I think of the idea and the concept behind sites and having the capability to run multiple sites on just one cms this seems to be just find.
Anything wrong with setting a completely different, in that sense unrelated domain in this config?

**Comment by jackmcdade** [comment]
Nope, do it! You might also be able to do `/` instead of a domain and have it work anywhere.

**Comment by Buffalom** [comment]
Not sure if I understand you correctly. The url of the site `default` was set to `/` by default. This did not work because it pointed to the root of the cms domain. Since I'm running the frontend and the cms on different domains this does not work.

**Comment by jackmcdade** [comment]
Ah sure, makes sense.

**Comment by christie-hill-za** [comment]
> Thanks guys, solved it all. 😄 👍

Hey @Buffalom can you please show how you solved it? I'm also running Statamic in headless mode and need to implement this for my users. Thanks!

**Comment by Buffalom** [comment]
> Hey @Buffalom can you please show how you solved it? I'm also running Statamic in headless mode and need to implement this for my users. Thanks!

In our site in `config/statamic/sites.php` we added the following line. The env-variable `FRONTEND_URL` in turn was set to something like `https://your-frontend-domain` in `.env`.

```
            'url' => env('FRONTEND_URL', '/'),
```
